### PR TITLE
Adding changes for AAP4381 to the 2.2 branch

### PR DESCRIPTION
**Pull request by dcdacosta**
Bringing the changes for AAP-4381 into the version 2.2 branch. Original changes for the 2.3 branch were added through PR#980

Link to preview draft of 2.2 change (VPN required):  http://file.rdu.redhat.com/~ddacosta/AAP4381/master.html

See section 3.4.3. Example standalone automation hub inventory file

### CODE REVIEW

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+You must set the following six variables in your {PlatformName} installer inventory file to configure your {PrivateHubName} for LDAP authentication:
+
+* `automationhub_authentication_backend`
+* `automationhub_ldap_server_uri`
+* `automationhub_ldap_bind_dn`
+* `automationhub_ldap_bind_password`
+* `automationhub_ldap_user_search_base_dn`
+* `automationhub_ldap_group_search_base_dn`
+
+If any of these variables are missing, the Ansible Automation installer will  not complete the installation.
```
Comment (jself-sudoku) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:15]: Very minor; there's an extra space between "will" and "not". Found during peer review.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+When you configure your {PrivateHubName} with LDAP authentication, you must set the proper variables in your inventory files during the installation process.
+
+.Prerequisites (Add Prereqs to 2.2 only)
+
+* Ensure that your system is running {PlatformName} 2.2.1 or later.
+* Ensure that you are using {PrivateHubName} 4.5.2 or later.
+
+.Procedure
+
+. Access your inventory file according to the procedure in link:hhttps://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file].
```
Comment (jself-sudoku) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:29]: Another nit found. There's an extra "h" that needs to be removed here.
```suggestion
. Access your inventory file according to the procedure in link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.3/html-single/red_hat_ansible_automation_platform_installation_guide/index#proc-editing-installer-inventory-file_platform-install-scenario[Editing the {PlatformName} installer inventory file].
```

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 3]
```diff
@@ -0,0 +1,177 @@
...
+. If you plan to set up extra parameters in  your {PrivateHubName} (such as user groups, super user access, mirroring, or others), proceed to the next section.
+
+
+== Configuring extra LDAP parameters
+
+If you plan to set up super user access, user groups, mirroring or other extra parameters, you can create a yaml file that comprises them in your `ldap_extra_settings` dictionary.
+
+.Procedure
+
+. Create a yaml file that will contain `ldap_extra_settings`, such as the following:
```
Comment (jself-sudoku) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:66]:
```suggestion
. Create a YAML file that will contain `ldap_extra_settings`, such as the following:
```

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+-----
+
+====
+
+. If you plan to set up extra parameters in  your {PrivateHubName} (such as user groups, super user access, mirroring, or others), proceed to the next section.
+
+
+== Configuring extra LDAP parameters
+
+If you plan to set up super user access, user groups, mirroring or other extra parameters, you can create a yaml file that comprises them in your `ldap_extra_settings` dictionary.
```
Comment (jself-sudoku) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:62]:
```suggestion
If you plan to set up super user access, user groups, mirroring or other extra parameters, you can create a YAML file that comprises them in your `ldap_extra_settings` dictionary.
```

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc [hunk 5]
```diff
@@ -0,0 +1,177 @@
...
+auth_ldap_user_search_scope= `SUBTREE'
+auth_ldap_user_search_filter= `(uid=%(user)s)`
+auth_ldap_group_search_scope= 'SUBTREE'
+auth_ldap_group_search_filter= '(objectClass=Group)`
+auth_ldap_group_type_class= 'django_auth_ldap.config:GroupOfNamesType'
+-----
+
+====
+
+. If you plan to set up extra parameters in  your {PrivateHubName} (such as user groups, super user access, mirroring, or others), proceed to the next section.
```
Comment (jself-sudoku) [inline downstream/modules/platform/ref-ldap-config-on-pah.adoc:57]:
```suggestion
. If you plan to set up extra parameters in  your {PrivateHubName} (such as user groups, superuser access, mirroring, or others), proceed to the next section.
```

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 1 above)
Comment (dcdacosta) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc]: Fixed.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc]: Fixed.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc]: Fixed.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 4 above)
Comment (dcdacosta) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc]: Fixed.

File: downstream/modules/platform/ref-ldap-config-on-pah.adoc (hunk 5 above)
Comment (dcdacosta) [reply downstream/modules/platform/ref-ldap-config-on-pah.adoc]: Fixed.

**Pull request merged**